### ISSUE TITLE
Pandas `groupby` not shown

### ISSUE THREAD

**Issue by nyngwang** [microsoft/pyright#3558]
**Describe the bug**
Coming from #735:

The `groupby` doesn't show up with pyright. This happened on pandas 1.4.2.

<img src="https://user-images.githubusercontent.com/24765272/170982153-74d932fa-a104-4186-a366-22a5459debdd.png" width="600">

**To Reproduce**
Try completion with pandas 1.4.2. (Read a file into dataframe from any .csv first.)

**Screenshots or Code**
above

If your code relies on symbols that are imported from a third-party library, include the associated import statements and specify which versions of those libraries you have installed.

**VS Code extension or command-line**
Are you running pyright as a VS Code extension or a command-line tool? Which version? You can find the version of the VS Code extension by clicking on the Pyright icon in the extensions panel.
neovim 0.8 master HEAD

**Additional context**

**Comment by erictraut** [comment]
Are you using any type stubs for pandas? If not, do you have `useLibraryCodeForTypes` enabled?

**Comment by nyngwang** [comment]
> Are you using any type stubs for pandas?

No. I did try it and there are some more serious problems regarding keyword params. (During the testing I disabled `use...`)

> [..] If not, do you have useLibraryCodeForTypes enabled?

Yes. This is my neovim config (ignore the syntax, but four options)

```lua
  settings = {
    -- https://github.com/microsoft/pyright/blob/main/docs/settings.md
    python = {
      analysis = {
        autoSearchPaths = true,
        diagnosticMode = "workspace",
        typeCheckingMode = 'off',
        useLibraryCodeForTypes = true,
      }
    }
  }
```

**Comment by erictraut** [comment]
If you're not using type stubs and enable the `useLibraryCodeForTypes`, then pyright will do its best to infer types from the source, but there are limits to type inference. There will be holes like the one you're seeing here. I encourage you to use type stubs.

If you find bugs in the `pandas-stubs` package, please report them to [that project](https://github.com/VirtusLab/pandas-stubs).

**Comment by nyngwang** [comment]
**update**: The problem is resolved on pandas version 1.5.0 (installed via conda-forge):

<img src="https://user-images.githubusercontent.com/24765272/191162304-fc174668-f9bb-4d59-b8d9-93e1b6c5a53b.png" width="600">
